metar.ts

```diff
@@ -39,6 +39,10 @@ const sendMetarMessage = async (metar: string, msg: Message) => {
 		embed.addField('Temperature :thermometer:', `${m.temperature.celsius}°C (${m.temperature.farenheit || Math.round(m.temperature.celsius * 9 / 5)}°F)`);
 		embed.addField('Dewpoint :droplet:', `${m.dewpoint.celsius}°C (${m.dewpoint.farenheit || Math.round(m.dewpoint.celsius * 9 / 5)}°F)`);
 		embed.addField('Altimeter :chart_with_upwards_trend:', `${m.altimeter.inches} inHg (QNH ${m.altimeter.millibars})`);
+		var da = calcDA(m.station,m.temperature.celsius,m.altimeter.inches);
```

Comment:
Please use `const` or `let` instead of `var`

Reply:
Done